File: src/sdk-reference/java/1/collection/create/snippets/create.java
try {
    String mapping = "{\"properties\": {\"license\": {\"type\": \"keyword\"}}}";

    kuzzle.getCollection().create("nyc-open-data", "yellow-taxi", mapping);

    System.out.println("Success");
} catch (KuzzleException e) {
    System.err.println(e.getMessage());
}

File: src/sdk-reference/go/1/collection/create/snippets/create.go
mapping := json.RawMessage(`{"properties":{"license": {"type": "keyword"}}}`)
err := kuzzle.Collection.Create("nyc-open-data", "yellow-taxi", mapping, nil)

if err != nil {
  log.Fatal(err)
} else {
  fmt.Println("Success")
}
